<launch>

	<!-- Mico services: These services interface with moveit-->
	<node name="moveit_joint_pose_service" pkg="bwi_moveit_utils" type="moveit_joint_pose_service">
	    <param name="arm/solve_type" value="Manipulation2" />
	</node>
	<node name="moveit_cartesian_pose_service" pkg="bwi_moveit_utils" type="moveit_cartesian_pose_service">
	    <param name="arm/solve_type" value="Manipulation2" />
	</node>

	<node name="moveit_waypoint_service" pkg="bwi_moveit_utils" type="moveit_waypoint_service">
        <param name="arm/solve_type" value="Manipulation2" />
    </node>

</launch>
